### PR TITLE
set operation

### PR DESCRIPTION
**Pull request by Flunavgt**
On this PR the following was applied:

Set operations on the calculator.
Created logic folder with files calculate and operate.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,42 +1,67 @@
...
+    super(props);
+    this.state = {
+      total: '',
+      next: '',
+      operation: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [x] Please remove all linters disabled lines and fix linters issues one by one.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,42 +1,67 @@
...
+      <button type="button" onClick={props.handler} className="btnPad3">3</button>
+      <button type="button" onClick={props.handler} className="btnPad4">4</button>
+      <button type="button" onClick={props.handler} className="btnPad5">5</button>
+      <button type="button" onClick={props.handler} className="btnPad6">6</button>
+      <button type="button" onClick={props.handler} className="btnPad7">7</button>
+      <button type="button" onClick={props.handler} className="btnPad8">8</button>
+      <button type="button" onClick={props.handler} className="btnPad9">9</button>
+      <button type="button" onClick={props.handler} className="btnPadAdd">+</button>
+      <button type="button" onClick={props.handler} className="btnPadMinus">-</button>
+      <button type="button" onClick={props.handler} className="btnPadDiv">+</button>
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [x] This button suppose to be for division calculation `÷` not addition `+` so please fix that to make the calculator work as expected.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Flunavgt 

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please remove unused sections in your readme file and add a proper description of how to clone your app and run it locally in the `Getting started` section to make it more professional.

image

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Flunavgt,

Good job on making the changes asked by the previous reviewer. 👍 

Your project is complete!  

## Highlights:

✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ App works fine

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**